src/main.rs

```diff
@@ -397,14 +406,17 @@ fn encode_token(matches: &ArgMatches) -> JWTResult<String> {
                 .collect(),
             _ => panic!("Invalid JSON provided!"),
         });
-    let expires = PayloadItem::from_string_with_name(matches.value_of("expires"), "exp");
+    let now = Utc::now().timestamp();
+    let expires = PayloadItem::process_timestamp(matches.value_of("expires"), "exp", now);
+    let not_before = PayloadItem::process_timestamp(matches.value_of("not_before"), "nbf", now);
+    let issued_at = PayloadItem::from_string_with_name(Some(now.to_string().as_str()), "iat");
```

Comment:
`now.to_string().as_str()` should be able to be written as `&now.to_string()`. Also, let's use `from_timestamp_with_name`, even though the timestamp is static and force all timestamp items to go through that function

Reply:
Thanks for the tip, applied that shorthand in two places, looks better!